src/rabbit_cli.erl

```diff
@@ -125,9 +135,22 @@ main(ParseFun, DoFun, UsageMod) ->
             rabbit_misc:quit(?EX_SOFTWARE)
     end.
 
+start_distribution_anon(0, LastError) ->
+    {error, LastError};
+start_distribution_anon(TriesLeft, _) ->
+    NameCandidate = list_to_atom(rabbit_misc:format("rabbitmq-cli-~2..0b", [rabbit_misc:random(100)])),
```

Comment:
One can run `make package-generic-unix UNOFFICIAL_RELEASE=true` from the umbrella to produce a tarball. It should take a couple of minutes.